sdk/resourcemanager/azure-resourcemanager-containerinstance/src/main/java/com/azure/resourcemanager/containerinstance/models/ContainerGroup.java

```diff
@@ -92,32 +160,54 @@ public interface ContainerGroup extends GroupableResource<ContainerInstanceManag
     DnsNameLabelReusePolicy autoGeneratedDomainNameLabelScope();
 
     /**
+     * Gets the id of the subnets..
+     *
      * @return the id of the subnets.
      */
     List<ContainerGroupSubnetId> subnetIds();
 
-    /** @return whether managed service identity is enabled for the container group */
+    /**
+     * Whether managed service identity is enabled for the container group.
+     *
+     * @return whether managed service identity is enabled for the container group
+     */
     boolean isManagedServiceIdentityEnabled();
 
     /**
+     * Gets the tenant id of the system assigned managed service identity.
+     *
      * @return the tenant id of the system assigned managed service identity. Null if managed service identity is not
      *     configured.
      */
     String systemAssignedManagedServiceIdentityTenantId();
 
     /**
+     * Gets the principal id of the system assigned managed service identity.
+     *
      * @return the principal id of the system assigned managed service identity. Null if managed service identity is not
      *     configured.
      */
     String systemAssignedManagedServiceIdentityPrincipalId();
 
-    /** @return whether managed service identity is system assigned, user assigned, both, or neither */
+    /**
+     * Whether managed service identity is system assigned, user assigned, both, or neither.
```

Comment:
Checks

Reply:
fixed by the [commit#2a564fb](https://github.com/Azure/azure-sdk-for-java/pull/43516/commits/2a564fbcb9e103076243b7eaf7c11d47ae22f596)